src/views/Tasks/index.vue

```diff
@@ -1,14 +1,239 @@
+    <span class="red--text">"body"</span> : <span class="red--text">"The body of the article"</span>
+    <span class="red--text">"author"</span> : <span class="red--text">"The author"</span>
+  }
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+  
+        </div>
+  
+      <div id="get-all-articles">
+      <h3 class=title>Get all articles</h3>
+        <v-divider></v-divider>
+        <br />
+  
+        <p>A GET request on route <code>"/articles"</code> should return all the articles stored in the database
+          If there are no articles in the database, then the request should return an empty array.
+          An example of the GET request is shown below:
+        </p>
+  
+        <v-card >
+          <v-card-subtitle>
+            GET /articles response:
+          </v-card-subtitle>
+          <v-card-text>
+            <pre>
+  [            
+    {
+      <span class="red--text">"_id"</span> : <span class="red--text">"ID"</span>
+      <span class="red--text">"title"</span> : <span class="red--text">"The Title"</span>
+      <span class="red--text">"subtitle"</span> : <span class="red--text">"The subtitle"</span>
+      <span class="red--text">"body"</span> : <span class="red--text">"The body of the article"</span>
+      <span class="red--text">"author"</span> : <span class="red--text">"The author"</span>
+    },
+    ...
+  ]
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+        </div>
+  
+        <div id="get-by-id">
+     <h3 class=title>Get a specific article by ID</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>A GET request on route <code>"/articles/ID"</code> should return the article with the respective ID.
+        If the article does not exist, then the request should return error 404.
+        </p>
+  
+        <v-card >
+          <v-card-subtitle>
+            GET /articles/ID response:
+          </v-card-subtitle>
+          <v-card-text>
+            <pre>          
+  {
+    <span class="red--text">"_id"</span> : <span class="red--text">"ID"</span>
+    <span class="red--text">"title"</span> : <span class="red--text">"The Title"</span>
+    <span class="red--text">"subtitle"</span> : <span class="red--text">"The subtitle"</span>
+    <span class="red--text">"body"</span> : <span class="red--text">"The body of the article"</span>
+    <span class="red--text">"author"</span> : <span class="red--text">"The author"</span>
+  }
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+        </div>
+  
+        <div id="update-article">
+        <h3 class=title>Update article</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>A PUT request on the <code>"/articles/ID" </code> route shall replace the entry corresponding to the article's id in the database.
+           If the given article is invalid (i.e. the body, title, subtitle or lead paragraph is either empty), the route shall return a 400.
```

Comment:
1. 400 status code should only happen when the id in the request isn't an ObjectID
2. instead of "return a 400", use "respond with a status code 400"